app/services/search/postgres/user.rb

```diff
@@ -0,0 +1,74 @@
+module Search
+  module Postgres
+    class User
+      ATTRIBUTES = %i[
+        id
+        name
+        profile_image
+        username
+      ].freeze
+
+      DEFAULT_PER_PAGE = 60
+      MAX_PER_PAGE = 100
+
+      # User.search_sore used to take employer related fields into account, but they have since been moved to profile
+      # and removed from fields that are searched against.
+      HOTNESS_SCORE_ORDER = Arel.sql(%{
+        (((articles_count + comments_count + reactions_count + badge_achievements_count) * 10) * reputation_modifier)
+        DESC
+      }.squish).freeze
+
+      def self.search_documents(term: nil, sort_by: :nil, sort_direction: :desc, page: 0, per_page: DEFAULT_PER_PAGE)
+        # NOTE: we should eventually update the frontend
+        # to start from page 1
+        page = page.to_i + 1
+        per_page = [(per_page || DEFAULT_PER_PAGE).to_i, MAX_PER_PAGE].min
+
+        relation = ::User
+
+        relation = filter_suspended_users(relation)
+
+        relation = relation.search_users(term) if term.present?
+
+        relation = relation.select(*ATTRIBUTES)
+
+        relation = sort(relation, sort_by, sort_direction)
```

Comment:
I did [something similar to this for comments](https://github.com/forem/forem/pull/13401/files#diff-7a9e59611c79676c66dec5f95924a3d85d41684b84f9e58a8b18ac062f9dcf3fR81). I'm thinking we can worry about unifying everything once the migration is done.